frontend/src/components/common/table/SearchableTablePage.tsx

```diff
@@ -36,12 +36,14 @@ const SearchableTablePage = <T, SortPropTypes extends readonly string[]>({
   );
   return (
     <>
-      <VStack pt={4} spacing={6} w="full">
-        <HStack width="100%">
-          {searchBarComponent}
-          {sortMenuComponent}
-          {filterMenuComponent}
-        </HStack>
+      <VStack py={4} spacing={6} w="full">
+        {!noResults && (
```

Comment:
Just to make sure, have you checked that we never pass in `filteredData.length` for this prop anywhere? That would result in filters breaking the page, so you can probably just set the filters on every filter page to a combination with no results.